### ISSUE TITLE
Bug Report: Opponent stops singing on miss when bf has no vocals file

### ISSUE THREAD

**Issue by IDiMooo** [FunkinCrew/Funkin#2837]
## Describe the bug

When there is only one vocal file for opponent and not for boyfriend, this audio will stop play on miss even if it's not bf's vocal file.

## To Reproduce

Create a song with only one vocal file for opponent and miss a note before opponent's turn.

## Expected behavior

The audio should still play if I understand why the vocals were seperated.

## Screenshots/Video

![image](https://github.com/FunkinCrew/Funkin/assets/163159763/762eb5e5-6b59-4996-abbe-d8db988ba77f)

https://github.com/FunkinCrew/Funkin/assets/163159763/60069027-27c1-4983-82a7-02741da4d09a

## Desktop
 - OS: Windows 10
 
 - Browser: nope
 
 - Version: 0.4.1
 

## Additional context

I was porting some mods and they have the old Voices files so I just wanted to give it to opponent so it won't mute and the miss sounds will be the only thing but I noticed that even if the voices are on the opponent's side they still get muted after miss. If both vocal files are there it mutes just bf's but if there is one for opponent it mutes it too. I don't know if it is intented but ye.

**Comment by M7theguy** [comment]
This is the reason why they seperated the vocals. The mod you are playing only has one file for vocals of the opponent AND Boyfriend. A fix for this might be to check how many vocal files there are and shorten the miss effect

**Comment by IDiMooo** [comment]
Yeah I get why they seperated the files it's just the fact that bf can mute opponents side in this case which i think may not be intended
